src/js/components/Checkbox/index.js

```diff
@@ -0,0 +1,2 @@
+import Checkbox from './Checkbox'
+export default Checkbox
```

Comment:
eof

Reply:
Adding a specified linter with the spacing would fix your concerns here.  i don't think it's a big deal.  🤷‍♂️   But we can agree on a linter say standard, airbnb or another.